data/coalition/coalition weapons.txt

```diff
@@ -130,6 +132,8 @@ outfit "Active Finisher"
 		"radar tracking" .8
 		"optical tracking" .5
 		"missile strength" 50
+		"damage dropoff" 4000
+		"dropoff modifier" 1.3
```

Comment:
I'd provide a second value here. This submunition doesn't actually have a velocity line, meaning that the "range" is 0, so this could effectively read `"damage dropoff" 4000 0` (since not providing a second value is the same as the second value equaling the "range" of the weapon as calculated by `velocity * lifetime`), an instant jump from 100% to 130% damage after 4000.

For reference, the actual range of the active Finisher is 5000, but after #5515 it'll be 4982.

Reply:
ah, I assumed it'd gradually increase to 130% after 4000.
the range window for this damage dropoff on the Active Finisher is why I opened as a draft, as I'm not yet sure what that should be at.

the damage increase itself would be to fit into the Active Finisher is actually stronger than the parent projectile, a damage boost so that the damage nerf to the initial Finisher isn't just a flat nerf, which should also encourage better usage than just facetanking an enemy while firing Finishers as that'd result in less damage, whereas gaining distance before firing would mean you deal more damage than normal. For NPCs this'd be a nerf in most cases as they wouldn't be in many situations where they fire at the right distance to make use of this, but should make missile-boat variants more valuable.

for an in-universe explanation, I don't have one.
if I were to try to stretch and get something, observing the projectiles the initial Finisher glows brighter until it eventually turns into Active Finisher, which then will "blink", going a bit brighter and then back to normal, doing so 5 times before reaching the end of its range, all while leaving behind a trail as if propelling something, which is probably just a visual thing, at most showing off that the projectile is faster now, but could be said to be product of an enrichment process taking place inside the torpedo, regarding its payload, and the moment it dies off being due to the reaction reaching a point of instability.

if you meant why specifically 4000 again wasn't really sure on what value to put there; if the above rambling makes more sense could have it increase from spawn to 4900?

Reply:
> ah, I assumed it'd gradually increase to 130% after 4000.

You can do that with `"damage dropoff" 4000 5000`. The only reason there's an issue with only providing one value is because this is a submunition, so the `lifetime * velocity` range calculation isn't correct at all.